odsApi_versioned_docs/version-7.3/getting-started/source-code-installation/readme.md

```diff
@@ -398,11 +398,11 @@ variable is properly configured.
 
 To set the "StandardVersion" variable:
 
-1. Press the **Windows key** 🪟 on your keyboard, type **Environment
+1. Press the **Windows key** on your keyboard, type **Environment
```

Comment:
You didn't like my use of the Windows emoji? 😢. I admit, I have wondered if it displays correctly on a Mac or Linux. But then again, it doesn't really matter if it doesn' display, since these are Windows instructions. This is how it renders in the website. I like having it, but I'll let go of the idea if you all prefer not to have it.

![image](https://github.com/user-attachments/assets/e154d423-0990-4bdb-9586-53efe3896e6f)

Reply:
Oops. It rendered an empty rectangle on my machine, and I incorrectly assumed it was a Confluence-specific icon lost in the conversion to Markdown. Now that I realize what it is, I'm happy to return them.